ci/jobs/README.md

```diff
@@ -0,0 +1,29 @@
+When defining jobs in this file, please organize files according to JJB's recommended layout:
+
+http://docs.openstack.org/infra/system-config/jjb.html#configuring-projects
+
+The summary of what this means:
+- Job defaults go in `defaults.yaml`
+- Macros go in `macros.yaml`
+- Templates go in in files named similarly to the jobs they create so we can easily find them
+- Jobs using those templates are defined in `projects.yaml`, and the name of the project should
+  match the name of the file containing that project's jobs.
```

Comment:
Worth mentioning to use `-` as separator for yaml files? This will ensure a pattern on the name. On robottelo-ci I see files using both `-` and `_` as separator and sometimes in the same file name.